cmd/agent/common/commands/check.go

```diff
@@ -60,6 +62,8 @@ var (
 	profileMemoryFilters string
 	profileMemoryUnit    string
 	profileMemoryVerbose string
+
+	checkOutput bytes.Buffer
```

Comment:
Do we need to make it a global var ? It's global for all command if defined here. And I think we could pass it to each function, no ?